### PR TITLE
Chem Heater gets a new name

### PR DESCRIPTION
**Pull request by Robertha89**
Chemical Heating Ventilation Cooling Unit or a Chemical HVC, due to the
unit not being connecting to any pipes, due to funding cuts, it
ventilates everything directly into the room.

### CODE REVIEW

File: code/modules/reagents/newchem/chem_heater.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /obj/machinery/chem_heater
-	name = "chemical heater"
+	name = "chemical hvc unit"
+	desc = "It's a Heating Venting Cooling unit for chemicals."
```
Comment (tigercat2000) [inline code/modules/reagents/newchem/chem_heater.dm]: I feel like this should have dashes between Heating, Venting, and Cooling.

**Comment by Iamgoofball** [comment]
This seems useless
:-1:

**Comment by Robertha89** [comment]
it is fluff essentially, and clarity, cause now they can cool, so calling it an chemical heater would be wrong, and people wanted a new name, so here ya go.

**Comment by veganzombeh** [comment]
:-1: The name "Chemical Heater" makes the function of the machine pretty clear. This name doesn't.

**Comment by Fox-McCloud** [comment]
:-1: 

I have a floor heater that can technically cool the room; doesn't mean it has to be given some fancy name.

**Comment by ZomgPonies** [comment]
:-1:

**Pull request closed without merging**